config/owasp/suppressions.xml

```diff
@@ -2234,6 +2234,13 @@
     </suppress>
     <suppress>
         <notes><![CDATA[
+    file name: htmlunit-cssparser-1.5.0.jar
+    ]]></notes>
+        <packageUrl regex="true">^pkg:maven/net\.sourceforge\.htmlunit/htmlunit\-cssparser@.*$</packageUrl>
+        <cve>CVE-2020-5529</cve>
+    </suppress>
+    <suppress>
+        <notes><![CDATA[
```

Comment:
suppressing css parser as it is the latest version, so upgrade is not an option.